plugins/router/template/router.go

```diff
@@ -139,9 +165,7 @@ func newTemplateRouter(cfg templateRouterCfg) (*templateRouter, error) {
 		return nil, err
 	}
 	glog.V(4).Infof("Committing state")
```

Comment:
This is the startup code path. But that said, I'll add a couple more log messages in the rate limited commit function (so that there's a trackback).